app/deck/install/binary.md

```diff
@@ -4,23 +4,31 @@ title: decK Binary
 
 decK is distributed as a Go binary for Windows, MacOS, and Linux. Releases are published to [GitHub Releases](https://github.com/Kong/deck/releases).
 
-## Windows
+## Installation
+
+### Windows
 
 To install decK on Windows, download the release artifact ending with `_windows_amd64.tar.gz` and extract it. Next, run `./deck.exe` from your terminal.
 
-## MacOS
+### MacOS
 
 Kong provides a [Homebrew](https://brew.sh) tap for decK. To install decK on MacOS, run:
 
 ```bash
 brew install kong/deck/deck
 ```
 
-## Linux
+### Linux
 
 Download the `.deb` or `.rpm` file from the releases page and install it using your package manager.
 
-## Mise
+## Upgrade deck
+
+To upgrade deck, follow the instructions above again for the new version. This will upgrade your installed version.
```

Comment:
This might be true for most of the install options, but with homebrew you can do `brew upgrade deck`, which is what I normally do.

Reply:
`brew upgrade deck` upgrades everything. `brew install deck` will upgrade deck if an upgrade is available

Reply:
TIL :)